API/OAI_Specs.yaml

```diff
@@ -132,21 +132,143 @@ paths:
+                  resources:
+                    "Title 1": 3 # title -> usage
+                    "Title 2": 2
+                - id: 4
+                  title: Project 2
+                  description: This is for project 2
+                  users:
+                    - Frank
+                    - Ayan
+                  creator: Ayan
+                  resources:
+                    "Title 1": 3 # title -> usage
+                    "Title 2": 2
+        '401':
+          description: Unauthorized. JWT token is missing or invalid.
+          content:
+            application/json:
+              example:
+                msg: User not found
+    post:
+      summary: Create a new project for a user
+      security:
+        - JWTAuth: []
+      tags: 
+        - Projects
+      requestBody:
+        description: Project data
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                title:
+                  type: string
+                  description: Project title
+                description:
+                  type: string
+                  description: Project description
+    
+      responses:
+        '200':
+          description: Project created successfully
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/Projects'
+                type: object
+                properties:
+                  projectId:
+                    type: integer
+                    description: ID of the created project
+        '401':
+          description: Unauthorized. JWT token is missing or invalid.
+          content:
+            application/json:
+              example:
+                msg: User not found
+        '500':
+          description: Internal server error. Failed to create the project.
+          content:
+            application/json:
+              example:
+                msg: error message
+  /projects/{project_title}/users:
+    post:
+      summary: "Add a new user to the project"
+      description: "Add a new user to the specified project."
+      tags:
+        - Projects
+      security:
+        - JWTAuth: []
+      parameters:
+        - name: "project_title"
+          in: "path"
+          description: "Name of the project to which the user will be added"
+          required: true
+          schema:
+            type: "string"
+          example: "Project 1"
+      # requestBody:
+      #   required: false
+      #   content:
+      #     application/json:
+      #       schema:
+      #         type: "object"
+      #         properties:
+      #           userId:
+      #             type: "string"
+      #             description: "Name of the user to be added to the project"
+      #             example: "Mimi" # not required, if not provided, add yourself
```

Comment:
could we remove this

Reply:
This is included in the API spec, but you're not required to send the requestBody. So to avoid confusion, I commented it out.